src/viam/services/slam/slam.py

```diff
@@ -71,7 +71,7 @@ async def get_position(self, *, timeout: Optional[float]) -> Pose:
         ...
 
     @abc.abstractmethod
-    async def get_properties(self, *, timeout: Optional[float]) -> Tuple[bool, MappingMode.ValueType]:
+    async def get_properties(self, *, timeout: Optional[float]) -> Tuple[bool, MappingMode.ValueType, str, List[SensorInfo]]:
```

Comment:
Ok so this is technically a breaking change, since anyone who is unpacking the tuple will now get an error

```python3
x, y = await slam.get_properties().  # ValueError: too many values to unpack (expected 2)
```
Also, this tuple is getting kinda long. Since we're basically just returning the `GetPropertiesResponse` but destructured, I think we should actually use a `TypeAlias` here and simply return the response as is. We do that already in a few places (see [camera](https://github.com/viamrobotics/viam-python-sdk/blob/9ec4ab4993101cfe793d239f2003b1c49a15982a/src/viam/components/camera/camera.py#L38)).

It could look something like:

```python3
# Other imports
# ...

if sys.version_info >= (3, 10):
    from typing import TypeAlias
else:
    from typing_extensions import TypeAlias


class SLAM(ServiceBase):
    ...

    Properties: "TypeAlias" = GetPropertiesResponse

    ...

    async def get_properties(self, *, timeout: Optional[float]) -> SLAM.Properties:
         ...
```

Then, in the client, we can simply return the response as is in both the client and service.

Reply:
Done - let me know if it looks correct!